### PR TITLE
Remove lambda, zetaw and zetas from states

### PR DESCRIPTION
**Pull request by finsberg**
In the master branch we have lambda, zetas and zetaw as state variables in the EP model. This means that the variables are also changed when we solve the EP model and this might be problematic. Here we pass them in as function to the Cellmodel contstructor.

### CODE REVIEW

File: src/simcardems/em_model.py [hunk 1]
```diff
@@ -83,7 +83,4 @@ def mechanics_to_coupling(self):
 
     def coupling_to_ep(self):
```
Comment (IlsevanHerck) [inline src/simcardems/em_model.py:84]: Do we need to keep this function? It seems that we're not updating the EP anymore explicitly.

**Review by IlsevanHerck — COMMENTED** [review]
Great catch and fix! 
Initial results with this code look promising :)

File: src/simcardems/em_model.py (hunk 1 above)
Comment (finsberg) [reply src/simcardems/em_model.py:84]: We could remove it because it is not really doing anything, but I don't think it does any harm either

Comment (jorgensd) [reply src/simcardems/em_model.py:84]: Calling functions has a tiny cost, even if they dont do anything;)

**Comment by IlsevanHerck** [comment]
It is not intuitive to me when we update parameters such as `zetas`, `zetas_mech` and `zetas_prev`. However, rewriting this still gives the same simulation results.

**Pull request merged**